app/views/users/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1>User Details</h1>
+<%= render partial: "user_card" , locals: { user: @user } %>
+  <%= render partial: "bio_card" , locals: { user: @user } %>
+
+    <h2>Last three posts:</h2>
+
+    <div class="card-list">
+      <% @recent_posts.each do |post| %>
+        <%= render partial: "posts/post_card" , locals: { post: post } %>
+          <% end %>
+    </div>
+
+    <%= link_to "See all posts" , user_posts_path(@user) if @user.posts_counter> 0 %>
+      <%= link_to "Add a new post" , new_user_post_url(@user) %>
+
+        <%=form_with model: User.new, url: destroy_user_session_path, method: :delete, data: { turbo: false }  do %> <button class="custom-button">Sign out</button>
+          <% end %>
```

Comment:
![image](https://user-images.githubusercontent.com/65068771/179354775-1a87c4f8-8317-4e5e-a787-e409ca625736.png)

- [ ] Kindly fix this error accruing on this page.